cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -167,7 +167,11 @@ public ExecutableModellingResult transformToExecutable(ParsedSlang parsedSlang,
                 errors.addAll(validateFlowResultsHaveNoExpression(results, execName));
 
                 executableDependencies = fetchDirectStepsDependencies(workflow);
-                systemPropertyDependencies = dependenciesHelper.getSystemPropertiesForFlow(inputs, outputs, results, workflow.getSteps());
+                try {
+                    systemPropertyDependencies = dependenciesHelper.getSystemPropertiesForFlow(inputs, outputs, results, workflow.getSteps());
+                } catch (RuntimeException ex) {
+                    errors.add(ex);
+                }
```

Comment:
shouldn't the exception be added to the list also in the case of the OPERATION in this switch?